homeassistant/core.py

```diff
@@ -134,6 +134,7 @@
 BLOCK_LOG_TIMEOUT = 60
 
 ServiceResponse = JsonObjectType | None
+EntityServiceResponse = dict[str, ServiceResponse] | None
```

Comment:
I don't think we should include optional in `EntityServiceResponse`. That's confusing when reading the code where this is used. It's better to be explicit in every place about when something is optional.